src/components/molecules/SfGallery/README.md

```diff
@@ -0,0 +1,62 @@
+# SfGallery
+
+<!-- Write about general purpose of the component. Include screenshot (to be replaced with a live example once we migrate to vuepress) -->
+
+Swappable gallery for product images.
+
+![desktop](https://screenshotscdn.firefoxusercontent.com/images/0dec9565-4c31-45b1-9554-20b19fdf33c0.png)
+
+![desktop](https://screenshotscdn.firefoxusercontent.com/images/6234ea80-b523-4a68-9050-73d4c6ccf301.png)
+
+## Props
```

Comment:
we giot rid of those parts since they're autodocummented, just add proper comments on component for descriptions

Reply:
Nice, I had not seen the change in the template...
So, Just events, CSS modifiers and SCSS variables, right?

Reply:
Exactly. Check current template on master for ref